src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return dayRent - 20;
+  }
+
+  if (days >= 7) {
+    return dayRent - 50;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, you should create variables for main values - 20, 40, 50